spec/models/bucket_spec.rb

```diff
@@ -1,9 +1,10 @@
 require 'spec_helper'
 
-describe Bucket do
+RSpec.describe Bucket do
   let(:rack_request) { ActionController::TestRequest.new }
 
-  subject { described_class.create(name: 'test123') }
+  let(:bucket_name) { 'My Bucket' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.